tests/tests_management_commands.py

```diff
@@ -1,20 +1,50 @@
 from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+import logging
+
 from django.core.management import call_command
 
-from django_elastic_migrations.models import Index, IndexAction
+from django_elastic_migrations import DEMIndexManager, es_client
+from django_elastic_migrations.models import Index, IndexVersion
 from django_elastic_migrations.utils.test_utils import DEMTestCase
+from tests.es_config import ES_CLIENT
 from tests.models import Movie
-from tests.search import MovieSearchIndex, MovieSearchDoc
+from tests.search import MovieSearchIndex, MovieSearchDoc, get_new_search_index, alternate_textfield, DefaultNewSearchDocTypeMixin
+
+log = logging.getLogger(__file__)
+
+
+# noinspection PyUnresolvedReferences
+class CommonDEMTestUtilsMixin(object):
+
+    def _check_basic_setup_and_get_models(self, index_name='movies', expected_num_versions=1):
```

Comment:
a new `CommonDEMTestUtilsMixin` helper class has for now a single helper method `_check_basic_setup_and_get_models` for asserting that a particular index is set up as we expect and extracts frequently used variables.